doc/book/connectors/__python.rst

```diff
@@ -44,3 +44,214 @@ Also there are several community-driven Python connectors:
 * `asynctnt <https://github.com/igorcoding/asynctnt>`__ with asyncio support
 * `aiotarantool <https://github.com/shveenkov/aiotarantool>`__ also with asyncio support
 * `gtarantool <https://github.com/shveenkov/gtarantool>`__ with gevent support **no active maintenance**
+
+Feature comparison
+------------------
+
+Last update: December 2021
+
+..  list-table::
+    :header-rows: 1
+
+    *   -   Parameter
+        -   `igorcoding/asynctnt <https://github.com/igorcoding/asynctnt>`__
+        -   `shveenkov/aiotarantool <https://github.com/shveenkov/aiotarantool>`__
+        -   `shveenkov/gtarantool <https://github.com/shveenkov/gtarantool>`__
+        -   `tarantool/tarantool-python <https://github.com/tarantool/tarantool-python>`__
+
+    *   -   License
+        -   Apache License 2.0
+        -   LGPL
+        -   LGPL
+        -   Modified BSD
```

Comment:
I guess the Simplified BSD license (BSD-2-Clause) was meant and the Modified BSD license (BSD-3-Clause) was referred by a mistake. I would like to resolve this ambiguity: https://github.com/tarantool/tarantool-python/issues/197.

Reply:
Okay, I'll change it as soon as PR will be merged.

Reply:
Changed to BSD-2 as discussed.